modules/rosa-sdpolicy-am-aws-compute-types.adoc

```diff
@@ -315,6 +314,25 @@
 - g5.24xlarge (96 vCPU, 384 GiB)
 - g5.48xlarge (192 vCPU, 768 GiB)
 - dl1.24xlarge  (96 vCPU, 768 GiB)&#8224;
+- g6.xlarge (4 vCPU, 16 GiB)
+- g6.2xlarge (8 vCPU, 32 GiB)
+- g6.4xlarge (16 vCPU,	64 GiB)
+- g6.8xlarge (32 vCPU, 128 GiB)
+- g6.16xlarge (64 vCPU, 256 GiB)
+- gr6.4xlarge (16 vCPU, 128 GiB)
+- gr6.8xlarge (32 vCPU, 256 GiB)
```

Comment:
Is there a reason the two `gr6` instance types (currently lines 322 and 323) are mixed in with the `g6`s. I think all the `g6` types should be listed in order, then the `gr6`s and then the `g6e` to make it easier to read.